linux/datadog_wrapper

```diff
@@ -1,62 +1,79 @@
 #!/usr/bin/env sh
 
-CURRENT_DIR=$(pwd)
+main() {
+    # Required to add the AAS metadata
+    export DD_AZURE_APP_SERVICES=1
 
-# Required to add the AAS metadata 
-export DD_AZURE_APP_SERVICES=1
+    echo "Setting application directory as $APP_DIR"
+    APP_DIR=$(pwd)
+   
+    echo "Setting Datadog environment variables"
+    setEnvVars
 
-if [ -z "$DD_DIR" ]; then
-    DD_DIR="/home/datadog"
-fi
+    echo "Creating and switching to the Datadog directory at $DD_DIR"
+    mkdir -p $DD_DIR && cd $DD_DIR || return
 
-if [ -z "$DD_BINARIES_URL" ]; then
-    DD_BINARIES_URL="https://github.com/DataDog/datadog-aas-extension/releases/download"
-fi
+    if [ "$WEBSITE_STACK" = "NODE" ]; then
+        echo "Initializing setup for a Node environment"
+        setUpNodeEnv;
+    elif [ "$WEBSITE_STACK" = "DOTNETCORE" ]; then
+        echo "Initializing setup for a .NET environment"
+        setUpDotnetEnv;
+    fi
 
-if [ -z "$DD_AAS_LINUX_VERSION" ]; then
-    DD_AAS_LINUX_VERSION="linux-v0.1.3-beta"
-fi
+    echo "Executing start command: \"$DD_START_APP\""
+    cd "$APP_DIR" || return
```

Comment:
I would expect this directory to always exist and thus the `return` to never be hit.

Reply:
[Shellcheck](https://www.shellcheck.net/wiki/SC2164) is complaining when I don't have it which is why I added it.